src/selection/Selection.js

```diff
@@ -211,8 +222,14 @@ define(
                 context: context,
                 element: element
             };
+
+            if (!this.isSelectable(selectable)) {
```

Comment:
We already have access to the element, it would be simpler to just pass the element to your isSelectable function, instead of creating a variable called selectable only to return out of the function because the element is not selectable

Reply:
@deeptailor , fixed, thanks